infra/hooks/postdeploy.ps1

```diff
@@ -0,0 +1,13 @@
+# PowerShell syntax
+echo "Configuring EventGrid subscription for API Center"
+
+# Sleep for 2 minutes to allow the function app to be deployed
+Start-Sleep -s 120
+
+$output = azd env get-values
+$resourceGroupName = ($output -split "`n" | Where-Object { $_ -match 'RESOURCE_GROUP_NAME' }).Split("=")[1]
+$azureApiCenterId = ($output -split "`n" | Where-Object { $_ -match 'AZURE_API_CENTER_ID' }).Split("=")[1]
+$azureFunctionName = ($output -split "`n" | Where-Object { $_ -match 'AZURE_FUNCTION_NAME' }).Split("=")[1]
+$azureFunctionId = az functionapp function show --name $azureFunctionName --function-name apicenter-analyzer --resource-group $resourceGroupName --query "id" --output tsv
```

Comment:
Let's load these values from get-values into the process like we do in azure-search-openai-demo https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/scripts/load_azd_env.ps1